### PR TITLE
[3.9] Associations Toolbar button broken when editing category

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes
Same code exists for menu items. Was missing in category model

### Testing Instructions
Create multilingual site using sample data.
Edit a category tagged to a language.
Click on the Associations toolbar button
Instead of displaying the side by side of com_associations, it just saves the category and redirects to the manager.

Patch and test again.

### CODE REVIEW

**Comment by infograf768** [comment]
@HLeithner 
This is also for staging, i.e 3.9.17

Please merge urgent in both

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 279afae0f7781e92e9f8a23f5d892cd4ffdeb52b<br /><br />on staging<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28675">issues.joomla.org/tracker/joomla-cms/28675</a>.</sub>

**Comment by HLeithner** [comment]
can you rebase it on staging please? then it will be merged into 3.10 on release.

File: administrator/components/com_categories/models/category.php [hunk 1]
```diff
@@ -664,6 +665,7 @@ public function save($data)
 					return false;
 				}
 			}
+
```
Comment (richard67) [inline administrator/components/com_categories/models/category.php]:
```suggestion
```

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 279afae0f7781e92e9f8a23f5d892cd4ffdeb52b<br /><br />Tested with success on staging.

@HLeithner If @infograf768 can't rebase it for some reason, maybe you can just cherry pick his commits into staging?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28675">issues.joomla.org/tracker/joomla-cms/28675</a>.</sub>

**Comment by richard67** [comment]
@infograf768 I've suggested a little CS fix (unnecessary empty line removed).

**Comment by infograf768** [comment]
changed branch but some issues apparently

**Comment by infograf768** [comment]
This is now badly broken. Closing

**Comment by richard67** [comment]
New PR for staging is #28678 .

**Pull request closed without merging**